service/test/PlatformTopoTest.cpp

```diff
@@ -239,6 +240,31 @@ void PlatformTopoTest::SetUp()
         "NUMA node0 CPU(s):     3ffff00003ffff\n"
         "NUMA node1 CPU(s):     ffffc0000ffffc0000\n"
         "Flags:                 fpu vme de pse tsc msr pae mce cx8 apic sep mtrr pge mca cmov pat pse36 clflush dts acpi mmx fxsr sse sse2 ss ht tm pbe syscall nx pdpe1gb rdtscp lm constant_tsc arch_perfmon pebs bts rep_good nopl xtopology nonstop_tsc aperfmperf eagerfpu pni pclmulqdq dtes64 monitor ds_cpl vmx smx est tm2 ssse3 fma cx16 xtpr pdcm pcid dca sse4_1 sse4_2 x2apic movbe popcnt tsc_deadline_timer aes xsave avx f16c rdrand lahf_lm abm 3dnowprefetch epb cat_l3 cdp_l3 invpcid_single intel_pt spec_ctrl ibpb_support tpr_shadow vnmi flexpriority ept vpid fsgsbase tsc_adjust bmi1 hle avx2 smep bmi2 erms invpcid rtm cqm rdt_a rdseed adx smap xsaveopt cqm_llc cqm_occup_llc cqm_mbm_total cqm_mbm_local dtherm ida arat pln pts\n";
+    m_no_numa_lscpu_str =
+        "Architecture:          x86_64\n"
+        "CPU op-mode(s):        32-bit, 64-bit\n"
+        "Byte Order:            Little Endian\n"
+        "CPU(s):                72\n"
+        "On-line CPU(s) mask:   ffffffffffffffffff\n"
+        "Thread(s) per core:    2\n"
+        "Core(s) per socket:    18\n"
+        "Socket(s):             2\n"
+        "NUMA node(s):          2\n"
+        "Vendor ID:             GenuineIntel\n"
+        "CPU family:            6\n"
+        "Model:                 79\n"
+        "Model name:            Intel(R) Xeon(R) CPU E5-2695 v4 @ 2.10GHz\n"
+        "Stepping:              1\n"
+        "CPU MHz:               2101.000\n"
+        "CPU max MHz:           2101.0000\n"
+        "CPU min MHz:           1200.0000\n"
+        "BogoMIPS:              4190.38\n"
+        "Virtualization:        VT-x\n"
+        "L1d cache:             32K\n"
+        "L1i cache:             32K\n"
+        "L2 cache:              256K\n"
+        "L3 cache:              46080K\n"
+        "Flags:                 fpu vme de pse tsc msr pae mce cx8 apic sep mtrr pge mca cmov pat pse36 clflush dts acpi mmx fxsr sse sse2 ss ht tm pbe syscall nx pdpe1gb rdtscp lm constant_tsc arch_perfmon pebs bts rep_good nopl xtopology nonstop_tsc aperfmperf eagerfpu pni pclmulqdq dtes64 monitor ds_cpl vmx smx est tm2 ssse3 fma cx16 xtpr pdcm pcid dca sse4_1 sse4_2 x2apic movbe popcnt tsc_deadline_timer aes xsave avx f16c rdrand lahf_lm abm 3dnowprefetch epb cat_l3 cdp_l3 invpcid_single intel_pt spec_ctrl ibpb_support tpr_shadow vnmi flexpriority ept vpid fsgsbase tsc_adjust bmi1 hle avx2 smep bmi2 erms invpcid rtm cqm rdt_a rdseed adx smap xsaveopt cqm_llc cqm_occup_llc cqm_mbm_total cqm_mbm_local dtherm ida arat pln pts\n";
```

Comment:
This is the output used for the new unit test, would be better to have something more representative of the failing case.

Reply:
The failing system had two notable differences from the unit test above:
- `On-line CPU(s) list: 0-71` Was provided instead of 'On-line CPU(s) mask:'. 
- lscpu listed `CPU(s): 72` and `Core(s) per socket: 16`.  This means that for the above example `Cores*Sockets*Threads < CPU(s)`.  This triggered another exception in PlatformTopo, however this behavior seems reasonable given that lscpu is providing inconsistent data..  WIP to determine why this happens and if there's a reasonable workaround.